mmagic/models/editors/stable_diffusion/stable_diffusion_inpaint.py

```diff
@@ -0,0 +1,497 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, List, Optional, Union
```

Comment:
I found this implementation seems heavily referred to other repo's implementation. Please mention their copyright here.